packages/app/src/runner/SpecRunnerHeaderOpenMode.vue

```diff
@@ -63,11 +63,11 @@
         :disabled="autStore.isRunning"
       >
         <template #heading>
-          <img
+          <component
```

Comment:
This conditional logic is repeated in several places - can it be abstracted to a new component?